test/e2e/density.go

```diff
@@ -118,24 +118,6 @@ var _ = Describe("Density [Skipped]", func() {
 
 	// Gathers data prior to framework namespace teardown
 	AfterEach(func() {
-		// Remove any remaining pods from this test if the
```

Comment:
I'm not convinced about this change - this means that we won't be measuring latency of API calls deleting pods.
I think that we should revert this part of this PR.
Or actually even better, move deleting the RC to the test body.